sql/all_match_pairs_with_um.sql

```diff
@@ -1,44 +1,30 @@
 -- add "self matches" for the articles that didn't match anything (this can happen if the article has a lot of null
 -- fields) to the rest of the article match pairs
-WITH oa_matches AS (
```

Comment:
removing as we can no longer use mag to match (directly, anyway). For further discussion: https://github.com/georgetown-cset/article-linking/issues/29